android/demo/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -77,14 +85,15 @@ public void reportOverallStatus(String status, long time) {
         }
     }
 
-    Secrets mSecrets;
     // NetworkIntentService mService;
     boolean mBound = false;
     TextView mOutputTextView;
     int mUrlCount = 0;
 
-    String httpsUrl = "";
-    String envoyUrl = "";
+    String httpUrl = "";
```

Comment:
I'm not sure I remember why we had separate `envoyUrl` and `httpsUrl` values here, were we testing some feature of the `enovy://` URLs? Were we just testing the same URL encoded differently?

IIRC I recycled one of them for MASQUE because I didn't know how to update the UI or what we had both